web-client/src/styles/custom.scss

```diff
@@ -859,7 +859,7 @@ textarea#case-caption {
 }
 
 .view-pdf-button svg {
-  color: $color-white;
+  color: #005ea2;
```

Comment:
It looks like we use `#005ea2` in a few places. Is it worth making styling constant?